### DISCUSSION TITLE
Does minio support hot and cold separation?

### DISCUSSION THREAD

**Discussion by qiuyuyin** [minio/minio#16352] [Q&A] [answered]
For example, our team now needs to put new a file on minio, and after three months, this file will be automatically transferred to AWS Archive Access Tier for archiving (because AWS Archive Access Tier is cheaper)

**Comment by harshavardhana** [comment, marked as answer]
We do support that via object tiering 
- https://min.io/product/automated-data-tiering-lifecycle-management
- https://min.io/docs/minio/linux/administration/object-management/object-lifecycle-management.html

**Reply by qiuyuyin** [reply]
amazing! thanks
